### ISSUE TITLE
Ambiguous compilation error when semicolon is missing from import statement

### ISSUE THREAD

**Issue by raineorshine** [trufflesuite/truffle#316]
## Issue

If you leave off the semicolon on an import statement, a pegjs error will be shown with the line number but not the filename in which the problem is occurring. This adds unnecessary troubleshooting steps for a trivial syntax error if there are many contracts in the project.

## Steps to Reproduce

Remove the semicolon from the end of the import statement in `MetaCoin.sol` and attempt to compile:

```js
import "./ConvertLib.sol"
```

## Expected Behavior

Should give a syntax error like browser-solidity:

```
MetaCoin.sol:5:1: Error: Expected token Semicolon got 'Contract'
contract MetaCoin {
^
```

## Actual Results

Compilation fails with no indication of which file the error is in:

```sh
/Users/raine/test/truffle-test/node_modules/solidity-parser/index.js:39
      throw e;
      ^
SyntaxError: Expected "/*", "//", ";", "as", comment, end of line or whitespace but "c" found. Line: 10, Column: 1
    at peg$buildException (/Users/raine/test/truffle-test/node_modules/solidity-parser/build/imports_parser.js:487:14)
    at Object.peg$parse [as parse] (/Users/raine/test/truffle-test/node_modules/solidity-parser/build/imports_parser.js:4080:13)
    at Object.module.exports.parse (/Users/raine/test/truffle-test/node_modules/solidity-parser/index.js:34:23)
    at /Users/raine/test/truffle-test/node_modules/truffle/lib/profiler.js:242:38
    at /Users/raine/test/truffle-test/node_modules/truffle/lib/sources.js:37:7
    at /Users/raine/test/truffle-test/node_modules/async/lib/async.js:789:30
    at /Users/raine/test/truffle-test/node_modules/async/lib/async.js:167:37
    at /Users/raine/test/truffle-test/node_modules/truffle/lib/sources.js:22:9
    at /Users/raine/test/truffle-test/node_modules/truffle/lib/sources/fs.js:15:12
    at tryToString (fs.js:449:3)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:436:12)
```

## Environment

* Operating System: OSX El Capitan
* Truffle version: 3.0.0-4
* Ethereum client: N/A
* node version: 6.1.0
* npm version: 3.8.6

**Comment by tcoulter** [comment]
Good catch, thanks @raineorshine

**Comment by gnidan** [comment]
This is probably not still a thing. Let me know if that's not true. Thanks!
